deploy/role.yaml

```diff
@@ -66,6 +66,21 @@ rules:
       - "list"
       - "watch"
       - "update"
+  - apiGroups:
```

Comment:
you should revisit the existing roles from above and remove those pointing to the old apigroups.